README.md

```diff
@@ -33,6 +33,132 @@ Get the latest version [here](https://github.com/llorllale/cactoos-matchers/rele
 
 Java version required: 1.8+.
 
+## cactoos-matchers versus Hamcrest + JUnit
+
+cactoos-matchers | Hamcrest (static method) | Hamcrest (object) | JUnit
+------ | ------ | ------ | ------
+`Assertion` | `MatcherAssert.assertThat` | - | -
```

Comment:
I'm thinking about the link from each our matcher to the corresponding section of the readme.
@llorllale, what do you think?

Reply:
@dgroup I don't plan to detail every single matcher in the README. Doing so would make it unwieldy. The README is there as a summary.

Reply:
Ok, the table as a summary is fine.